.github/ISSUE_TEMPLATE/1-documentation.md

```diff
@@ -9,16 +9,16 @@ assignees: ''
 
 <!--
 
-Welcome!  Unfortunately not all documentation is perfect, and if you're opening
+Welcome!Unfortunately, not all documentation is perfect, and if you're opening
 a documentation issue we are interested in fixing it.  Please fill out the
-template below so that we can solve the problem more quickly; or, alternately,
+template below so that we can solve the problem more quickly; or, alternatively,
```

Comment:
Either alternately or alternatively is valid here but I am indifferent, so changing to `alternatively` is fine with me.